dali/test/python/test_operator_mfcc.py

```diff
@@ -139,11 +134,17 @@ def check_operator_mfcc_wrong_args(device, batch_size, input_shape,
 def test_operator_mfcc_wrong_args():
     batch_size = 3
     for device in ['cpu', 'gpu']:
-        for dct_type, norm, axis, n_mfcc, lifter, shape, msg in \
-            [(1, True, 0, 20, 0.0, (100, 100), "Ortho-normalization is not supported for DCT type I"),  # DCT-I ortho-normalization is not supported
-             (2, False, -1, 20, 0.0, (100, 100), "Provided axis cannot be negative"),  # axis out of bounds
-             (2, False, 2, 20, 0.0, (100, 100), "Axis [\d]+ is out of bounds \[[\d]+,[\d]+\)"),  # axis out of bounds
-             (10, False, 0, 20, 0.0, (100, 100), "Unsupported DCT type: 10. Supported types are: 1, 2, 3, 4"),  # not supported DCT type
-            ]:
+        for dct_type, norm, axis, n_mfcc, lifter, shape, msg in [
+            # DCT-I ortho-normalization is not supported
+            (1, True, 0, 20, 0.0, (100, 100),
+             "Ortho-normalization is not supported for DCT type I"),
+            # axis out of bounds
+            (2, False, -1, 20, 0.0, (100, 100), "Provided axis cannot be negative"),
+            # axis out of bounds
+            (2, False, 2, 20, 0.0, (100, 100), "Axis [\\d]+ is out of bounds \\[[\\d]+,[\\d]+\\)"),
+            # not supported DCT type
+            (10, False, 0, 20, 0.0, (100, 100),
+             "Unsupported DCT type: 10. Supported types are: 1, 2, 3, 4"),
+        ]:
```

Comment:
I'd aim for some consistency here (e.g. move the error message to the next line always)

Reply:
done